### PR TITLE
Fix a logical error in level verification

### PR DESCRIPTION
**Pull request by drbeh**
### Description
This PR fixes a logical error in level verification. It did not have impact on the functionality of the reader but was verifying the default level.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).

### CODE REVIEW

File: monai/data/image_reader.py [hunk 1]
```diff
@@ -845,8 +846,6 @@ def get_data(
         return patches, metadata
 
     def _check_level(self, img, level):
```
Comment (Nic-Ma) [inline monai/data/image_reader.py:848]: I feel maybe it's better to change this function to `_get_level_count()`, then compare `level` and `level_count` at the outside.
What do you think?

Thanks.

**Review by Nic-Ma — APPROVED** [review]

**Comment by drbeh** [comment]
PyPI is dow! We should run the test again when it's back.

File: monai/data/image_reader.py [hunk 2]
```diff
@@ -844,21 +847,21 @@ def get_data(
...
-    def _check_level(self, img, level):
-        level = self.level
+    def _get_max_level(self, img_obj):
+        """
+        Return the maximum number of levels in the whole slide image
+        Args:
+            img: the whole slide image object
 
-        level_count = 0
+        Returns:
```
Comment (Nic-Ma) [inline monai/data/image_reader.py:856]: I feel no need this `returns` part as it's exactly same as the doc-string.

Thanks.

**Review by Nic-Ma — APPROVED** [review]

**Pull request merged**